### ISSUE TITLE
Create a bettter "create stacks app" builder

### ISSUE THREAD

**Issue by aulneau** [fungible-systems/micro-stacks#120]
I created a basic `create ustx` cli based off `vite-create`, but in talking with @hstove it would be better to have some kind of CLI that would let folks scaffold out a more complete starter where you take different features. Initially, this is what I was thinking:

- pick a framework 
  - react
    - next.js
    - remix
    - SPA    
  - vue
    - nuxt
    - SPA
  - svelte
    - sveltekit
    - SPA
  - solid
- pick a language
  - JS
  - TS
- include sponsored tx api?
- clarinet wrapper

Basically, we need a CLI tool that can merge multiple different types of projects together.

**Comment by Aslemammad** [comment]
Great, how does merging work? cause that's the tricky part here!
